test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -5215,6 +5227,116 @@ public void testModifiedFolder() throws Exception {
         new Checkpoint(FEB_1970, folderId));
   }
 
+  private void testModifiedFolderModifiedDocumentsQuery(String query,
```

Comment:
I'm not making much sense of this name, especially as a generic helper. I think "testModifiedDocumentsQuery" should do the trick.